src/core.js

```diff
@@ -1775,6 +1796,12 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
         /* eslint-disable-next-line no-continue */
         continue;
 
+      } else if (i === 'className' || (i === 'tableClassName' && instance.table)) {
```

Comment:
Maybe it'd be better to split this condition, otherwise, you have to check `i` variable again to choose between `instance.table` and `instance.rootElement` elements.